src/applications/applicationList/animationSelect/index.tsx

```
@@ -0,0 +1,66 @@
+                <_.Board>
+                    <_.Rows>
+                        {
+                            (
+                                () => {
+                                    const filtered = DemoAnimation.animations.filter((animation) => {
+                                        return animation.name.trim().includes(name.trim());
+                                    });
+
+                                    return filtered.map((animation) => {
+                                        const element = <_.Item>
+                                                                <_.Image src={animation.image} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)